### PR TITLE
ADD: Added all the pages layout and theme.

### PR DESCRIPTION
**Pull request by zoef-yt**
[PREVIEW LINK](https://deploy-preview-2--jot-it-down-zoef.netlify.app/)
In this pr, I added

- [ ] Homepage
- [ ] Archivepage(layout)
- [ ] Trashpage(layout)
- [ ] Navigation using Navlink
- [ ] sidebar
- [ ] Theme changer

Files to check:- [ME](https://github.com/zoef-yt/jot-it-down/pull/2/files?file-filters%5B%5D=.css&file-filters%5B%5D=.js&file-filters%5B%5D=.jsx&show-viewed-files=false#diff-8c58db05b9e37d28023bcdaa4b05287798009271085d8115f3f2e91097395d1b)

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,16 +1,20 @@
 import React from 'react';
 import { Routes, Route } from 'react-router-dom';
-import { LandingPage, Header, Footer } from './Components/index';
+import { LandingPage, Header, Footer, NotesPage, Sidebar, ArchivePage, TrashPage } from './Components/index';
```
Comment (muskanbagrecha) [inline src/App.js]: You dont have to specify index here.

File: src/App.js [hunk 2]
```diff
@@ -1,16 +1,20 @@
...
 import './Components/utils/CSS/Common.css';
 
 function App() {
 	return (
 		<div className='app'>
 			<Header />
+			<Sidebar />
 			<Routes>
-				<Route path='/' element={<LandingPage />} />
+				<Route path='/landing-page' element={<LandingPage />} />
```
Comment (muskanbagrecha) [inline src/App.js:13]: Please add 404 route.

File: src/Components/Archive/ArchivePage.jsx [hunk 3]
```diff
@@ -0,0 +1,7 @@
+import React from 'react';
```
Comment (muskanbagrecha) [inline src/Components/Archive/ArchivePage.jsx:1]: Not required.

File: src/Components/Archive/ArchivePage.jsx [hunk 4]
```diff
@@ -0,0 +1,7 @@
...
+
+function ArchivePage() {
```
Comment (muskanbagrecha) [inline src/Components/Archive/ArchivePage.jsx]: Use arrow function.

File: src/Components/index.js [hunk 5]
```diff
@@ -1,3 +1,7 @@
 export { LandingPage } from './LandingPage/LandingPage';
 export { Header } from './Header/Header';
 export { Footer } from './Footer/Footer';
+export { NotesPage } from './NotesPage/NotesPage';
+export { Sidebar } from './Sidebar/Sidebar';
+export { ArchivePage } from './Archive/ArchivePage';
+export { TrashPage } from './Trash/TrashPage';
```
Comment (dev-prabhat) [inline src/Components/index.js:7]: I appreciate  this way of importing component in one file and export it from here

**Review by dev-prabhat — APPROVED** [review]

File: src/Components/Sidebar/Sidebar.jsx [hunk 6]
```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+import { NavLink } from 'react-router-dom';
+import './Sidebar.css';
+function Sidebar() {
+	return (
+		<ul className='body-left-nav'>
```
Comment (muskanbagrecha) [inline src/Components/Sidebar/Sidebar.jsx]: This should be div I think.

File: src/Components/Sidebar/Sidebar.jsx [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+				<LinkItem to='/archive' name='Archive' />
+				<LinkItem to='/trash' name='Trash' />
+			</ul>
+		</ul>
+	);
+}
+
+export { Sidebar };
+const LinkItem = (props) => {
+	const navLinkClass = ({ isActive }) => (isActive ? 'showcase-list-item showcase-list-item-active' : 'showcase-list-item');
```
Comment (muskanbagrecha) [inline src/Components/Sidebar/Sidebar.jsx]: showcase-list-item is common here so you can use string interpolation (concatenation).

File: src/Components/Sidebar/Sidebar.jsx [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+	return (
+		<li className='nav-li'>
+			<NavLink to={props.to} className={navLinkClass}>
```
Comment (muskanbagrecha) [inline src/Components/Sidebar/Sidebar.jsx]: Suggestion: destructure props

**Review by muskanbagrecha — CHANGES_REQUESTED** [review]
Please implement the suggested changes. Looks good otherwise.

**Review by iRohitGaur — APPROVED** [review]
UI looks good.

Nice work. Look into the suggestions that other gave. Rest is fine.

Good to merge 👍🏻

File: src/App.js (hunk 1 above)
Comment (zoef-yt) [reply src/App.js]: okay

File: src/App.js (hunk 2 above)
Comment (zoef-yt) [reply src/App.js:13]: Will be adding it in the next pr thanks for the suggestion.

File: src/Components/Archive/ArchivePage.jsx (hunk 4 above)
Comment (zoef-yt) [reply src/Components/Archive/ArchivePage.jsx]: done

File: src/Components/Sidebar/Sidebar.jsx (hunk 6 above)
Comment (zoef-yt) [reply src/Components/Sidebar/Sidebar.jsx]: done.

File: src/Components/Sidebar/Sidebar.jsx (hunk 8 above)
Comment (zoef-yt) [reply src/Components/Sidebar/Sidebar.jsx]: done

File: src/Components/Sidebar/Sidebar.jsx (hunk 7 above)
Comment (zoef-yt) [reply src/Components/Sidebar/Sidebar.jsx]: done

**Review by muskanbagrecha — APPROVED** [review]
Good to merge as you have implemented the suggested changes.

**Pull request merged**